### DISCUSSION TITLE
Issue compiling executable from fortran or using a pre-compiled executable

### DISCUSSION THREAD

**Discussion by yoerga** [phydrus/phydrus#52] [Compiling Fortran Code]
Hello,

If I download the fortran code for the executable, and add gfortran from MINGW to the directory, when I run make it names a "source_code-main\\hydrus.exe", but the executable does not appear in that directory. No error message is displayed.

Alternatively, if I download the Hydrus 4.08 executable and point to it in phydrus, I receive the following error: "Error opening file PCPINFOR.SYS" and a follow on dialogue saying the file may be missing or corrupted, but the file is actually present in the directory. I have seen mention of this on the Hydrus forum but no resolution. I am running Windows 10. 

Has anyone encountered similar issues and found workarounds to either approach?

Thanks!

**Comment by martinvonk** [comment]
Have you tried run make without a folder name? Such that it appears in the current working directory?
